8by8-org/web-app
# Iss158 update vote reg comp

## Overview 
> PR to address #158 and #160 updates to Voter Registration Completed form. Implements links to state online registration website or directions for registering by mail and links to voter registration resources.

## Test Plan 
> REQUIRED 
> Register to vote as a player from state with online voting and a challenger from a state without online voting and followed below:
1. Check all links on `/voterreg/completed` and modals associated with `/completed` lead to expected paths.
2. Verify voter reg form is emailed when the appropriate link is clicked.


## Follow ups 
> #161 Implement links to state site, get registration form, and reg form emailed modal on actions page.
